bin/wp-merge-to-cp.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# First parameter is the changeset number
+# Second parameter is the WP branch
+
+find_wp_remote=$(git remote -v | grep WordPress/wordpress-develop | tail -n1 | awk '{print $1;}')
+if [ -z "$find_wp_remote" ]; then
+    echo "Adding WP remote"
+    # Download WordPress dev
+    git remote add wp https://github.com/WordPress/wordpress-develop > /dev/null 2>&1
+    find_wp_remote='wp'
+fi
+
+find_cp_remote=$(git remote -v | grep ClassicPress/ClassicPress | tail -n1 | awk '{print $1;}')
+if [ -z "$find_cp_remote" ]; then
+    echo "Adding CP remote"
+    # Download WordPress dev
+    git remote add cp git@github.com:ClassicPress/ClassicPress.git > /dev/null 2>&1
+    find_cp_remote='cp'
+fi
+
+echo "Fetch CP/WP git repository"
+git fetch "$find_wp_remote" > /dev/null 2>&1
+git fetch "$find_cp_remote" > /dev/null 2>&1
+
+# Switch to ClassicPress branch
+echo "Sync your local repo"
+git fetch origin > /dev/null 2>&1
+echo "Switch to develop branch"
+git checkout develop > /dev/null 2>&1
```

Comment:
This should be `git checkout $find_cp_remote/develop` instead, and the script should not touch the `origin/develop` branch (many people edit their `develop` branch directly when making PRs, even though they shouldn't).